setup.tpl

```diff
@@ -13,11 +13,12 @@ mkdir -p /opt/vault || true
 
 ip_address="$(ip addr show eth0 | perl -n -e'/ inet (\d+(\.\d+)+)/ && print $1')"
 
-cat <<'EOF' > /etc/vault.d/vault.hcl
+cat >/etc/vault.d/vault.hcl <<EOF
```

Comment:
Enclosing the heredoc delimiter in `'` prevents it from expanding parameters, in this case `$ip_address`.

For example:
```
cat <<'EOF'
$SHELL
EOF
```
results in `$SHELL`, while
```
cat <<EOF
$SHELL
EOF
```
results in `/bin/zsh`.